app/views/images/index.html.erb

```diff
@@ -1,13 +1,13 @@
-<%= stylesheet_link_tag "image" %>
-
-<h1>Welcome!</h1>
-
-<%= link_to 'New image', new_image_path %>
-
-<table class="image_table">
-  <% @images.each do |image| %>
-      <tr>
-        <td><%= image_tag image.imageurl, class: "index_image" %></td>
-      </tr>
-  <% end %>
-</table>
+<div class="album">
+  <div class="container-fluid">
+    <div class="row align-items-center">
+      <% @images.each do |image| %>
+        <a href=<%= "#{images_path}/#{image.id}" %>>
+          <div class="card">
+            <%= image_tag image.imageurl, class: "card-img mx-auto my-auto" %>
+          </div>
+        </a>
+      <% end %>
+    </div>
+  </div>
+</div>
```

Comment:
Refactor your html to not use as many ```divs``` . You can combine some of your divs to include multiple classes...I would refactor this as such:
```
<div class="album--container container-fluid">
    <div class="row align-items-center card">
        <% @images.each do |image| %>
            <%= link_to image do %>
                <%= image_tag image.imageurl, class: "card-img mx-auto my-auto" %>
            <% end %>
        <% end %>
    </div>
</div>
```

Reply:
Makes sense.